tests/regression/runner.py

```diff
@@ -376,6 +374,69 @@ def test_find_transactions_by_tag(self):
 
         eval_stat(time_cost, "find transactions by tag")
 
+    def test_get_transactions_object(self):
+        logging.debug(
+            "\n================================get transactions object================================"
+        )
+        # cmd
+        #    0. 81 trytes transaction hash
+        #    1. 20 trytes transaction hash
+        #    2. 100 trytes transaction hash
+        #    3. unicode transaction hash
+        #    4. Null transaction hash
+        rand_tag = gen_rand_trytes(27)
+        rand_msg = gen_rand_trytes(30)
+        rand_addr = gen_rand_trytes(81)
+        post_data = {
+            "value": 0,
+            "message": rand_msg,
+            "tag": rand_tag,
+            "address": rand_addr
+        }
+        post_data_json = json.dumps(post_data)
+        sent_transaction_obj = API("/transaction/", post_data=post_data_json)
+
+        logging.debug("sent_transaction_obj = " +
+                      ", sent_transaction_obj[0] = " +
+                      sent_transaction_obj[0] +
+                      ", sent_transaction_obj[1] = " + sent_transaction_obj[1])
+        sent_transaction_obj_json = json.loads(sent_transaction_obj[0])
+        sent_transaction_hash = sent_transaction_obj_json["hash"]
+        test_cases = [
+            sent_transaction_hash, sent_transaction_hash[0:19],
+            sent_transaction_hash + gen_rand_trytes(19), "工程師批哩趴啦的生活", ""
+        ]
+
+        response = []
+        for t_case in test_cases:
+            logging.debug("testing case = " + repr(t_case))
+            response.append(API("/transaction/", get_data=t_case))
+
+        for i in range(len(response)):
```

Comment:
Is this for loop necessary?

Reply:
This loop can print all the response at once.
The same logging below is for showing the current examine test case.

Reply:
Sorry, I just feel like this can be done in a single loop. Could you re-evaluate?

Reply:
I think I need to print at least the current index at line 421. If not, debugging would be kinda tricky.